llvm/docs/CommandGuide/llvm-objdump.rst

```diff
@@ -291,6 +296,15 @@ OPTIONS
        cmp eax, dword ptr <g>
        jge	<L0>
 
+.. option:: --pretty-pgo-analysis-map
+
+  When using ``--symbolize-operands`` with bb-address-map and pgo-analysis-map,
```

Comment:
```suggestion
  When using :option:``--symbolize-operands`` with bb-address-map and pgo-analysis-map,
```
Also applies below - this will ensure the reference becomes a hyperlink to the actual option in the documentation.

Reply:
Thanks for the suggestion!